packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -135,18 +135,20 @@ export const Table = styled.table<{ $showFooter?: boolean }>`
     }
 
     .sticky-column {
-        position: sticky !important;
         left: 1px;
         z-index: 1;
-        background-color: white !important;
         word-break: break-word;
         :hover {
             white-space: normal;
             background-color: white;
         }
     }
+    th.sticky-column {
+        background: white !important;
```

Comment:
t headers, if sticky, should all be white. 
<img width="122" alt="image" src="https://github.com/user-attachments/assets/bf916de5-04c6-46c2-8ca9-b439d88e896c" />